src/functional/ffront/func_to_foast.py

```diff
@@ -42,6 +42,34 @@ class FieldOperatorSyntaxError(DialectSyntaxError):
     dialect_name = "Field Operator"
 
 
+class UnrollPowerOp(eve.NodeTranslator):
```

Comment:
```suggestion
class UnrollPowerOp(eve.NodeTranslator):
    @classmethod
    def apply(cls, node: foast.LocatedNode):
        return cls().visit(node)
```
We use the convention of adding an `apply` classmethod to all visitors. In this particular case it ensures that the type checker, i.e. mypy, know that this visitor is expecting an foast node. Afterwards you can use the visitor as so:
```
UnrollPowerOp.apply(my_foast_node)
```